terraform/scripts/manually-upload-certs.sh

```diff
@@ -2,8 +2,8 @@
 
 set -eu
 
-export PASSWORD_STORE_DIR=${CERT_PASSWORD_STORE_DIR}
-
+export PASSWORD_STORE_DIR=~/.paas-pass-high
```

Comment:
> This is incompatible with this naming convention because it needs the directory to end -pass.

Inconsistencies everywhere 🙈  Any thoughts about what it should be called? Do people use the alias with the credentials-high repo and where do you have it cloned to, @alext @timmow @HenryTK @bleach?

> I think it would be better to still allow overriding this with an environment variable (a proper one as opposed to make's approximation of them.).

How about (pending the directory name above) ?
```
export PASSWORD_STORE_DIR=${PASSWORD_STORE_DIR:-~/.paas-pass-high}
```